# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/CommonPropertyNames.h
# Commit: d9fa28df9787
# Full Hash: d9fa28df9787b4d01135df632c987f20e830b9d6
# Author: Jonatan Klemets <jonatan.r.klemets@gmail.com>
# Date: 2021-12-02 09:42:49
# Regressor Bug: 1736060
# File Overlap Count: 1
# Description:
#   Bug 1736060 - Part 5: Implement import assertions for static imports. r=mgaudet,arai
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D126045
# ==============================================================================

diff -r 5b9622ccd0d2 -r d9fa28df9787 js/src/vm/CommonPropertyNames.h
--- a/js/src/vm/CommonPropertyNames.h	Wed Dec 01 18:26:09 2021 +0000
+++ b/js/src/vm/CommonPropertyNames.h	Wed Dec 01 18:26:10 2021 +0000
@@ -39,6 +39,7 @@
   MACRO_(ArrayType, ArrayType, "ArrayType")                                    \
   MACRO_(ArrayValues, ArrayValues, "$ArrayValues")                             \
   MACRO2(as, as, "as")                                                         \
+  MACRO_(assert, assert_, "assert")                                            \
   MACRO_(Async, Async, "Async")                                                \
   MACRO_(async, async, "async")                                                \
   MACRO_(AsyncFromSyncIterator, AsyncFromSyncIterator,                         \